tests/repo/shallow.c

```diff
@@ -37,3 +39,39 @@ void test_repo_shallow__clears_errors(void)
 	cl_assert_equal_i(0, git_repository_is_shallow(g_repo));
 	cl_assert_equal_p(NULL, giterr_last());
 }
+
+void test_repo_shallow__shallow_oids(void)
+{
+	git_oidarray oids, oids2;
```

Comment:
Grr 😆. Any chance I could graft [`leaks`](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/leaks.1.html) on clar's exit, so I get a nice reminder when that happens ?